src/main/java/uk/gov/hmcts/reform/prl/services/ManageOrderEmailService.java

```diff
@@ -416,6 +421,8 @@ private void handlePersonalServiceNotifications(String authorisation, CaseData c
         if (C100_CASE_TYPE.equalsIgnoreCase(caseTypeOfApplication)) {
             nullSafeCollection(caseData.getApplicants()).stream().findFirst().ifPresent(party -> {
                 dynamicDataForEmail.put("name", party.getValue().getRepresentativeFullName());
+                log.info("CA personal service email notifications: handlePersonalServiceNotifications: dynamicDataForEmail: {}",
```

Comment:
Please remove this logger if its printing any PII data.

Reply:
The log has been removed